DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,121 @@
+# Developer Guide
+
+The developer guide is an on-boarding document for new contributors to FileOnQ.Imaging.Raw. This project implements some non-standard practices and the goal of this document is to answer questions and explain how the code works.
+
+## Compiling/Building
+
+TBD
+
+### Native Assemblies
+
+TBD
```

Comment:
We will be moving native assemblies to follow a similar pattern as heif. This means fileonq_libraw32.dll and fileonq_libraw.dll will be renamed to just libraw.dll. Once we do that we will use the runtimes folder structure
```
runtimes/
├─ win-x64/
│  ├─ libraw.dll
├─ win-x86/
│  ├─ libraw.dll
```

The example above is missing our CUDA assembly, that is just an example